LiskHQ/lisk-hub
# Add network switcher to settings - Closes #830

### What was the problem?
Not implemented network switcher
### How did I fix it?
Implemented it
### How to test it?
1. Go to settings page and change switcher to ON, return to login page and see if network settings are enabled.
2. Make sure setting page and settings switcher are accessible when not login
3. Make sure showNetwork works with url parameters
### Review checklist
- The PR solves #830
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
